### PR TITLE
Add external files support

### PR DESCRIPTION
**Pull request by soarer1004**
Add external files support. For example:

```html
<script lang="ts" src="./bar.ts"></script>
```

### CODE REVIEW

**Comment by pyoner** [comment]
Hi @soarer1004 !
Can you make the PR without external dependencies `path`, `fs`?

**Comment by soarer1004** [comment]
@pyoner 

Can avoid usage of `fs`, but i don't know how get path of script file without `path`.

File: packages/preprocess/package.json [hunk 1]
```diff
@@ -100,11 +100,13 @@
     "coveralls": "^3.0.2",
     "cross-env": "^5.2.0",
     "cz-conventional-changelog": "^2.1.0",
+    "fs": "^0.0.1-security",
     "husky": "^1.3.1",
     "jest": "^24.7.1",
     "jest-config": "^24.7.1",
     "lint-staged": "^8.0.0",
     "lodash.camelcase": "^4.3.0",
+    "path": "^0.12.7",
```
[the remark this answers is not in the document]
Comment (soarer1004) [reply packages/preprocess/package.json]: @pyoner 

Removed

**Comment by pyoner** [comment]
@soarer1004 
thank you!

**Comment by pyoner** [comment]
@soarer1004 
published https://www.npmjs.com/package/@pyoner/svelte-ts-preprocess/v/1.2.0

**Pull request merged**